### PR TITLE
Improved the English text for the auto-evo exploring tool

### PR DESCRIPTION
**Pull request by hhyyrylainen**
also fixed one string with a trailing newline Poedit complained about each time

**Related Issues**

translators, pretty fairly, complained that the English text was not clear at all in the auto-evo exploring tool, so I tried to improve that without getting too wordy

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [ ] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: locale/en.po [hunk 1]
```diff
@@ -1032,75 +1032,75 @@ msgstr "Viewer"
...
 
 #: ../src/auto-evo/AutoEvoExploringTool.tscn:308
 msgid "NEW_BIODIVERSITY_INCREASING_SPECIES_POPULATION"
-msgstr "New biodiversity increasing species population"
+msgstr "Initial population for biodiversity increasing species"
 
 #: ../src/auto-evo/AutoEvoExploringTool.tscn:322
 msgid "PROTECT_MIGRATIONS_FROM_SPECIES_CAP"
-msgstr "Protect migrations from species cap"
+msgstr "Protect just migrated populations from species cap"
```
Comment (84634E1A607A) [inline locale/en.po:1079]: Is "species cap" "force extinction"? If so, I think directly say "from force extinction" is better.

Comment (hhyyrylainen) [reply locale/en.po:1079]: I don't want to constraint people wanting to change the auto-evo too much by being overtly specific here.
I don't particularly like forced extinctions as a concept so I'd like us to move away from that in terms of the species cap, so I'd prefer to leave this as a more general way to say this.

Comment (hhyyrylainen) [reply locale/en.po:1079]: I'll merge this to get these changes on weblate for translators. If there's some better wording (that isn't too much longer as we need to be careful with these as these don't have line wrapping enabled) a suggestion can be made on weblate.
